version/description

```diff
@@ -1,12 +1,7 @@
-v0.12.0
-
-Features:
-* Add multi-platform support for macvtap cni image builds.
-* cluster: Allow overriding kubevirtci params
-* cluster: Pin kubevirtci tag
-* feat: add snyk file to exclude vendor folders
-* build: use golang 1.20
+v0.12.2
 
+Bugs:
+* Construct the image name directly within the build target
 ```
-docker pull quay.io/kubevirt/macvtap-cni:v0.12.0
+docker pull macvtap-cni:v0.12.2
```

Comment:
@ashokpariya0 can you please take a look if it is related to your changes ?
(the fact we miss the prefix)

Reply:
@oshoval Yeah, I checked that, and I don’t think it’s related to my change. Based on [prepare-release.sh](https://github.com/kubevirt/macvtap-cni/blob/4fbcfedeb854f5fa4c7abf405275a5148f423fef/hack/prepare-release.sh#L20), it seems like the image name is being prepared manually.

@maiqueb Any thoughts on this?

Reply:
thanks
maybe worth to add there a syntax check if it is indeed injected manually please ?
(but of course let make sure first as you said)
anyhow this is not high priority of course